manifests/get_32bit_agent.pp

```diff
@@ -0,0 +1,33 @@
+# -------
+# Fetch and unzip 32bit agent installer
+# -------
+
+class bootstrap::get_32bit_agent(
+  $version   = 'latest',
+  $architecture   = $::architecture,
+  $file_cache     = '/vagrant/file_cache'
+) {
+  $puppet_dir   = '/opt/puppet'
+  $repo_dir     = "${puppet_dir}/packages"
+  $public_dir   = "${repo_dir}/public"
+  $version_dir  = "${public_dir}/${version}"
+  $agent_dir    = "puppet-enterprise-${version}-el-${operatingsystemmajrelease}-i386-agent"
+  $agent_file   = "${agent_dir}.tar.gz"
+  $url          = "https://s3.amazonaws.com/pe-builds/released/${version}"
+
+  file { [$puppet_dir,$repo_dir,$public_dir,$version_dir]:
+    ensure => directory
+  }
+  staging::deploy { $agent_file:
+    source  => "${url}/${agent_file}",
```

Comment:
Ok, it looks like the latest thing doesn't work on the main URL either, so I'm just going to set the default to 3.7.2